### DISCUSSION TITLE
Are there any nice examples of combining sccache with cibuildwheel on Github Actions?

### DISCUSSION THREAD

**Discussion by hauntsaninja** [pypa/cibuildwheel#1519] [Q&A]
https://cibuildwheel.readthedocs.io/en/stable/working-examples/ mentions sccache in the etebase-py entry, but it seems like they haven't actually used sccache in years

**Comment by Mause** [comment]
Looks like https://github.com/pypa/cibuildwheel/issues/1030#issuecomment-1691408281 has it working?

**Comment by mwestphal** [comment]
I've not been able to make it work. Not every links work in the comment linked above.
